pkgs/tools/security/yubikey-touch-detector/default.nix

```diff
@@ -33,9 +33,12 @@ buildGoModule rec {
   '';
 
   postInstall = ''
-    install -Dm444 -t $out/share/doc/${pname} *.md
+    install -Dm444 -t $out/share/doc/${pname} *.{md,example}
+
+    install -Dm444 -t $out/share/licenses/${pname} LICENSE
 
     install -Dm444 -t $out/share/icons yubikey-touch-detector.svg
+    install -Dm444 -t $out/share/icons/hicolor/128x128/apps yubikey-touch-detector.png
```

Comment:
should the package switch to using the newly added icon? i'm having my reservations on this project being able to redistribute the yubico logo as part of their own asset.

Reply:
Yes, I think it makes sense to change to included icon.

Reply:
I have removed the SVG, does everything look good?